CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ enable_testing()
 option(RC_ENABLE_TESTS "Build RapidCheck tests" OFF)
 option(RC_ENABLE_EXAMPLES "Build RapidCheck examples" OFF)
 
+set(cmake_config_files
```

Comment:
Because I follow the "write-once" principle in programming. I'm going to add a version file as soon as you respond to #123 (if you say yes). If not I'll remove it as it's unnecessary. In short it's a container for the loop below.